regexMatches.js

```diff
@@ -288,11 +288,15 @@ module.exports = [
     },
   },
   {
-    regex: /#(\d{4})\b/,
+    // If contains either only PR references or some Git Hub related words
+    regex: /^(#\d{1,5})(\s+#\d{1,5})*$|\b(prs?|git(\s?hub)?|pull(\s?request)?s?|gh|issues?|commits?|merges?)\b/i,
```

Comment:
I think just removing the number check should resolve it as you've added a check below
```suggestion
    regex: /|\b(prs?|git(\s?hub)?|pull(\s?request)?s?|gh|issues?|commits?|merges?)\b/i,
```
Otherwise need the number at the start and end I think _(maybe)_
```suggestion
    regex: /\b((#\d{1,5}).*(prs?|git(\s?hub)?|pull(\s?request)?s?|gh|issues?|commits?|merges?)|(prs?|git(\s?hub)?|pull(\s?request)?s?|gh|issues?|commits?|merges?).*(#\d{1,5}))\b/i,
```

Reply:
Fair, people will just need to add a keyword in the message to ensure the bot is called.
I initially let that in case people wanted to just point to the PR while discussing about its changes (like it happened a few times). I thought people would not randomly type a #number and post, lol.